Makefile

```diff
@@ -22,17 +22,10 @@ reset-db:
 setup:
 	poetry install --with setup,community --verbose
 	poetry run python3 src/backend/cli/main.py
-win-setup:
-	poetry install --with setup,community --verbose
-	poetry run python src/backend/cli/main.py
 lint:
 	poetry run black .
 	poetry run isort .
 first-run:
 	make setup
 	make migrate
 	make dev
-win-first-run:
```

Comment:
Why do we need to remove it? Windows deployments depend on it.

Reply:
Good catch